cmake/TuttleMacros.cmake

```diff
@@ -111,20 +111,40 @@ function(tuttle_ofx_plugin_target PLUGIN_NAME)
         # Plugin target is a shared library
         add_library(${PLUGIN_NAME} MODULE ${PLUGIN_SOURCES})
 
+        # Get plugin type
+        string(FIND ${CMAKE_CURRENT_SOURCE_DIR} "image/io" IS_IOPLUGIN)
+        string(FIND ${CMAKE_CURRENT_SOURCE_DIR} "image/display" IS_DISPLAY_PLUGIN)
+        string(FIND ${CMAKE_CURRENT_SOURCE_DIR} "image/generator" IS_GENERATOR_PLUGIN)
+        string(FIND ${CMAKE_CURRENT_SOURCE_DIR} "image/process" IS_PROCESS_PLUGIN)
+        string(FIND ${CMAKE_CURRENT_SOURCE_DIR} "param/analysis" IS_ANALYSIS_PLUGIN)
+        string(FIND ${CMAKE_CURRENT_SOURCE_DIR} "param/debug" IS_DEBUG_PLUGIN)
+
+        # Add this new plugin to custom Makefile target
+        add_dependencies(ofxPlugins ${PLUGIN_NAME})
+        if(IS_IOPLUGIN GREATER -1)
+            add_dependencies(ofxIO ${PLUGIN_NAME})
```

Comment:
I don't know... This is the tuttle build system, used to build the tuttle plugins: the function already contains specific things about tuttle. Here with few diffs we can add custom Makefile targets for all of our plugins. And if an external ofx plugin is built with this method, it won't be added to one of the new Makefile targets defined here.